deploy-agent/deployd/common/utils.py

```diff
@@ -29,6 +29,7 @@
 
 log = logging.getLogger(__name__)
 
+TELEFIG_BINARY = "configure-serviceset"
```

Comment:
ideally this can go in some configuration file to collect hardcoded values

Reply:
otherwise there will be a mysterious error years down the road when this changes

Reply:
Curious how putting in a config file could avoid errors? Basically, we need to sync the current telefig bin with the code in this repo. Whether it stays here or in a config file, I don't see much difference unless there is an auto sync mechanism

Reply:
The point is not config, the point is having in a discoverable location. Hardcoded values strewn throughout the repo causes headaches especially from external references, in this case the telefig repo.  Better to have all in one place, the natural place is the configuration.

Reply:
by the way, this is an internal service, so it might be good to consider not having any reference to it here in the open source repo

Reply:
yeah great point. let me double check on that.